addons/info/src/components/Story.js

```diff
@@ -384,7 +384,7 @@ Story.propTypes = {
   showInline: PropTypes.bool,
   showHeader: PropTypes.bool,
   showSource: PropTypes.bool,
-  styles: PropTypes.func.isRequired,
+  styles: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
```

Comment:
Looks like this proptype may be left as is (we actually pass a function anyway)

Reply:
👍  done